flink-ml-lib/src/main/java/org/apache/flink/ml/classification/logisticregression/LinearModelData.java

```diff
@@ -37,60 +38,68 @@
 import java.io.OutputStream;
 
 /**
- * Model data of {@link LogisticRegressionModel}.
+ * Model data of {@link LogisticRegressionModel}, {@link FtrlModel}.
  *
  * <p>This class also provides methods to convert model data from Table to Datastream, and classes
  * to save/load model data.
  */
-public class LogisticRegressionModelData {
+public class LinearModelData {
 
     public DenseVector coefficient;
+    public long modelVersion;
```

Comment:
The versioning mechanism is different from that in `OnlineKMeans`. Shall we adopt the same practice across both online algorithms?